docs/lib/predictions/fragments/android/identify-celebrity.md

```diff
@@ -0,0 +1,50 @@
+## Set up your backend
+
+If you haven't already done so, run `amplify init` inside your project and then `amplify add auth` (we recommend selecting the *default configuration*).
```

Comment:
Doesn't the CLI prompt you to do this? Shouldn't the setup guide walk through this fundamental step? @wooj2 @TrekSoft

Reply:
Not sure if i fully understand your question.  Are you saying if you try to run `amplify add auth` before running `amplify init`, doesn't it tell you to run amplify init?  If that is your question, then yes.  i see:
```bash
$ amplify add auth
You are not working inside a valid amplify project.
Use 'amplify init' in the root of your app directory to initialize your project with Amplify
Error: You are not working inside a valid amplify project.
Use 'amplify init' in the root of your app directory to initialize your project with Amplify
...
...
```

Reply:
Ah, sorry no, I meant:

Doesn't the CLI prompt you to add auth when you add predictions if it is not yet configured?

Reply:
you are right! If `auth` isn't yet enabled while adding predictions, then CLI will prompt you to configure it also. However, I directly used the documentation from iOS page for simplicity. Should I change it?

Reply:
Up to you and @wooj2

Reply:
leaving it as is for now, since both are technically valid ways to add auth. I think it makes sense to add it at the same time as predictions, but this "Set up your backend" portion is present in every page...